### PR TITLE
Courses Table Component and Tests

### PR DESCRIPTION
**Pull request by ShivaanshK**
This PR implements the Courses Table component, the storybook for Courses Table, and unit tests for Courses Table.

# Issues Addressed

This PR addresses issue #9

### CODE REVIEW

File: frontend/src/main/components/Courses/CoursesTable.js [hunk 1]
```diff
@@ -0,0 +1,56 @@
+import React from "react";
+import { useTable, useSortBy } from 'react-table'
+import { Table, Button } from "react-bootstrap";
+import OurTable from "../OurTable";
+
+export default function CoursesTable({ courses }) {
+
+
+    courses = courses.map(course => {
+        course.description = course.description.slice(0, 100) + '...'
```
Comment (Peake2) [inline frontend/src/main/components/Courses/CoursesTable.js:10]: If the description is less than 100 characters, it won't be cut but will still get a '...'

File: frontend/src/main/components/Courses/CoursesTable.js [hunk 2]
```diff
@@ -0,0 +1,56 @@
+import React from "react";
+import { useTable, useSortBy } from 'react-table'
```
Comment (Peake2) [inline frontend/src/main/components/Courses/CoursesTable.js:2]: Unused import

File: frontend/src/main/components/Courses/CoursesTable.js [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+import { Table, Button } from "react-bootstrap";
```
Comment (Peake2) [inline frontend/src/main/components/Courses/CoursesTable.js:3]: Unused import

File: frontend/src/tests/components/Courses/CoursesTable.test.js [hunk 4]
```diff
@@ -0,0 +1,83 @@
...
+    });
+
+    test("CoursesTable is testId", async () => {
+        const {getByTestId } = render(
+            <CoursesTable courses={coursesFixtures.threeCourses} />
+        );
+        await waitFor( ()=> expect(getByTestId("CoursesTable-header-quarter")).toBeInTheDocument() );
+    });
+
+    test("click on a header and a sort caret should appear", async () => {
```
Comment (Peake2) [inline frontend/src/tests/components/Courses/CoursesTable.test.js:26]: IDK if it was implemented by you, but testing sort by description on threeCourses on storybook does not sort it correctly

File: frontend/src/tests/components/Courses/CoursesTable.test.js [hunk 5]
```diff
@@ -0,0 +1,83 @@
+import { render, waitFor, fireEvent } from "@testing-library/react";
```
Comment (Peake2) [inline frontend/src/tests/components/Courses/CoursesTable.test.js:1]: fireEvent is unused import

**Review by Peake2 — CHANGES_REQUESTED** [review]
2 Small fixes and some unused imports

**Review by Peake2 — APPROVED** [review]
Review 2: LGTM

**Review by bzamora020 — APPROVED** [review]
Looks good to me!

**Comment by Peake2** [comment]
> Looks good to me!

Could you merge and assign points please? "The base branch restricts merging to authorized users."

**Comment by bzamora020** [comment]
Thanks! This is a 10 points issue

**Pull request merged**